### ISSUE TITLE
[Feature Request] allow exporting contacts without phone

### ISSUE THREAD

**Issue by soredake** [telegramdesktop/tdesktop#8002]
### Is your feature request related to a problem?
I want to telegram to support exporting contacts without phones, and list them like this:
![Untitled](https://user-images.githubusercontent.com/5204968/83903289-864e4280-a766-11ea-8f24-7e4d49ff107e.png)

**Comment by freemangilgamesh** [comment]
i agree and suggested allowing "import contacts" without a phone. I have my phone contact list and i want to add my friends.

**Comment by soredake** [comment]
That bot..........

**Comment by freemangilgamesh** [comment]
?

**Comment by soredake** [comment]
Again.

**Comment by foxyseta** [comment]
I believe it is now possible to do contact exports w/o being logged in on mobile. You just need to wait for 24 hours. At least, I got a service notification from Telegram telling me so (but I was logged in on mobile and did not want to wait for 24h hours).
